content/docs/intro/concepts/organizing-stacks-projects.md

```diff
@@ -197,9 +195,22 @@ service = core.v1.Service(..., ResourceOptions(provider=provider))
 {{% choosable language go %}}
 
 ```go
-// StackReference is not supported in Go currently.
-//
-// See https://github.com/pulumi/pulumi/issues/1614.
+import (
+  "fmt"
+
+  "github.com/pulumi/pulumi/sdk/go/pulumi"
+)
+
+func main() {
+  pulumi.Run(func(ctx *pulumi.Context) error {
```

Comment:
Should tabs be used for indentation so if someone copies/pastes, it will already formatted in the way one would expect for Go?

Reply:
This is consistent with the other code examples in other languages in this file. Not sure if we have some sort of editor config checked in, but this is the behavior of `tab` in vs code. 

`go fmt` will sort this out when users take the snipped anyway, so not too concerned about it.